packages/web-app-external/tests/unit/components/__snapshots__/ErrorScreen.spec.ts.snap

```diff
@@ -3,6 +3,7 @@
 exports[`The external app error screen component displays an icon and a paragraph 1`] = `
 <div class="uk-text-center">
   <ocicon-stub size="xxlarge" name="warning"></ocicon-stub>
-  <p data-msgid="Error when loading the application" data-current-language="en_US">Error when loading the application</p>
+  <h1 class="oc-text-lead" data-msgid="Error when loading the application" data-current-language="en_US">Error when loading the application</h1>
+  <!---->
```

Comment:
Would like to see a scenario where an error message is given and rendered in the ErrorScreen test!

Reply:
I'm not sure how to add tests... Can you explain/give some pointers?

Reply:
The test lives close to the component (https://github.com/owncloud/web/blob/master/packages/web-app-external/tests/unit/components/ErrorScreen.spec.ts) and should be pretty self-explanatory:
- `describe` steps (strings) to explain the context
- `it` steps (strings) to assert certain behavior
- components get mounted and then you can assert a certain DOM structure (snapshot tests, like in this case), check if user interactions trigger correct behavior (by e.g. mocking clicks) etc
- you can mount them passing props (in this case you'd want one snapshot test with and one without the errorMessage and check if the `<p>` tag gets rendered if an errorMessage is given

Feel free to get inspired by the other tests, usually the files app offers more diverse test scenarios: https://github.com/owncloud/web/tree/master/packages/web-app-files/tests/unit/components